README.md

```diff
@@ -19,50 +17,67 @@ A few Sphinx extensions are used as well:
+
+#### Medium
+Here you will install a basic html version of `pros-docs` This is adequate for most people
+To make this process easier. If you don't already have a basic understanding of how these work:       [git](https://rogerdudler.github.io/git-guide/), 
+                  [github](https://github.com/firstcontributions/first-contributions), 
+                  [sphinx](https://www.mageworx.com/blog/2016/05/sphinx-the-beginners-guide/), 
+                  [python3](https://wiki.python.org/moin/BeginnersGuide), you should go check them out before you start.
+ 
+ Clone the project, change the directory to `pros-docs`
+ ```sh
+git clone https://github.com/purduesigbots/pros-docs
+cd pros-docs
+```
 
-The requirements to build the docs site (with no modifications to anything but the
-source (`.rst`) files) are as follows:
+The `sphinx_rtd_theme` and `sphinx-tabs` folders within the documentation are
+[git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules). As a result,
+you need to initialize these submodules and pull them separately when updating the docs.
 
+```sh
+# Repeat this in both the sphinx_rtd_theme and sphinx-tabs directories
+> git submodule init
+> git submodule update
+```
+
+ Install These Dependancies:
 * Python3
 * Sphinx (pip package)
 * Sphinx-tabs (pip package)
 * Ablog (pip package)
 
-To install all of the dependencies for building the PROS Documentation, run
-
-```
-# Recommend working in a virtual environment, but the following 2 lines are optional
+We recommend working in a virtual environment, but the following 2 lines are optional
 > python3 -m venv ./docs-venv
 > . ./docs-venv/bin/activate
 
-> pip3 install -r requirements.txt
-```
+Whether or not you decide to use the python virtual environment install all of the dependencies for building the PROS Documentation, run:
 
-### Building the Docs
+```sh
+pip3 install -r requirements.txt
+```
+Note: to get out of the virtual environment run `deactivate`
 
 Once you have installed the requirements, building is very straightforward. Just
-run `make` in the root directory.
-
-### Git Submodules
+run `make` in the `pros-docs` directory.
 
-The `sphinx_rtd_theme` and `sphinx-tabs` folders within the documentation are
-[git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules). As a result,
-you need to initialize these submodules and pull them separately when updating the docs.
+Now, pull up a browser and open `pros-docs/build/index.html`
 
-```
-# Repeat this in both the sphinx_rtd_theme and sphinx-tabs directories
-> git submodule init
-> git submodule update
-```
+Great! Now you can edit the `.rst` files, make new files, and send a PR to change those files on the actual website.

```

Comment:
not sure if it's just me, but this line sounds a little patronizing

Reply:
I think you're right. I didn't even think about that lol.